core/src/main/scala/org/apache/spark/internal/config/package.scala

```diff
@@ -715,6 +716,21 @@ package object config {
       .checkValue(v => v > 0, "The max failures should be a positive value.")
       .createWithDefault(40)
 
+  private[spark] val UNSAFE_EXCEPTION_ON_MEMORY_LEAK =
+    ConfigBuilder("spark.unsafe.exceptionOnMemoryLeak")
+      .booleanConf
+      .createWithDefault(false)
+
+  private[spark] val UNSAFE_SORTER_SPILL_READ_AHEAD_ENABLED =
+    ConfigBuilder("spark.unsafe.sorter.spill.read.ahead.enabled")
+      .booleanConf
+      .createWithDefault(true)
+
+  private[spark] val UNSAFE_SORTER_SPILL_READER_BUFFER_SIZE =
+    ConfigBuilder("spark.unsafe.sorter.spill.reader.buffer.size")
+      .longConf
+      .createWithDefault(UnsafeSorterSpillReader.DEFAULT_BUFFER_SIZE_BYTES)
```

Comment:
What about creating with value `1024 * 1024` here? Then, it seems that we can remove `DEFAULT_BUFFER_SIZE_BYTES` in `UnsafeSorterSpillReader.java` file and reference `UNSAFE_SORTER_SPILL_READER_BUFFER_SIZE` instead?